test-end-to-end/README.md

```diff
@@ -88,9 +86,9 @@ adding from device, test downloading from another device.
 ### 5	Discussion board
 5.1 Teacher can add discussion board and topics to course
 
-5.2 Teacher can edit discussion board and topics
+5.2 Teacher can edit/remove discussion board and topics
 
-5.3 All users on course can post on discussion board, see posts from other users. Each topic and post shows last active date and number of replies.
+5.3 All users on course can post on discussion board, see posts from other users. Each topic and post shows last active date and number of replies. Posts should be editable/deletable by owners/admin accounts.
```

Comment:
Edit by owner, deletion by users with appropriate rights

Reply:
> Edit by owner, deletion by users with appropriate rights

Edit by owner (e.g. the original poster, and it would need to be flagged as edited), and deletion by users who have permission to edit the course (e.g. teacher or admin) makes sense to me 

It isn't part of any existing milestone. It wouldn't be a huge amount of additional work, and could be added to a milestone if the checklist is formally accepted.